src/main/java/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/BuildCardExtension.java

```diff
@@ -277,14 +325,17 @@ protected int triggerBuild(final AbstractProject<?, ?> triggerProject, final Abs
 
         List<AbstractBuildParameters> configs = null;
 
+
         final BuildPipelineTrigger manualTrigger = upstreamProjectPublishersList.get(BuildPipelineTrigger.class);
         if (manualTrigger != null) {
             LOGGER.fine("Found Manual Trigger (BuildPipelineTrigger) found in upstream project publisher list ");
             final Set<String> downstreamProjectsNames =
                     Sets.newHashSet(Splitter.on(",").trimResults().split(manualTrigger.getDownstreamProjectNames()));
+
             LOGGER.fine("Downstream project names: " + downstreamProjectsNames);
             // defect: requires full name in the trigger. But downstream is just fine!
-            if (downstreamProjectsNames.contains(project.getFullName())) {
+
+            if (downstreamProjectsNames.contains(project.getName())) {
```

Comment:
The previous case is not the issue - we correctly identify the project by full name because we require build pipeline trigger to be specified by full name (well, before the change above). The problem is on line line 299 (line 350 in your branch) -- this assumes that the BuildTrigger config is also being specified by full name. This is not a safe assumption - it is in fact always going to be specified by a relative name (i.e `admin-api/Prod Build/`), which we compare to our full name and the mismatch happens